apps/api/.spectral.yaml

```diff
@@ -0,0 +1,18 @@
+# Spectral is a flexible JSON/YAML linter and validator, which can be used for OpenAPI, AsyncAPI, or any other purpose.
+# To run Spectral locally:
+# > pnpm lint:openapi
+
+# For information on creating custom rulesets, see:
+# https://meta.stoplight.io/docs/spectral/01baf06bdd05a-create-a-ruleset
+
+# Base rulesets to extend from:
+extends: [[spectral:oas, all]]
+
+# Override rules from base ruleset. Useful to incrementally enable rules on legacy files.
+# https://meta.stoplight.io/docs/spectral/293426e270fac-overrides
+overrides:
+  - files:
+      - "**#/paths/~1v1~1subscribers~1%7BsubscriberId%7D~1preferences~1%7BtemplateId%7D"
+      - "**#/paths/~1v1~1subscribers~1%7BsubscriberId%7D~1preferences~1%7Blevel%7D"
```

Comment:
These endpoints have a different parameter for the same endpoint path. This is not RESTful and was caught by Spectral, see error here:
https://github.com/novuhq/novu/actions/runs/7114141317/job/19367636093#step:6:51

We need to deprecate the `:level` path and move to a new one. For now, this override prevents Spectral from evaluating the path as an error. I will create a ticket to address.